templates/airVisual/README.md

```diff
@@ -1,6 +1,10 @@
 Weather and Air Quality (AirVisual)
 ===================================
 
+![Static Badge](https://img.shields.io/badge/Download%20-%20Click%20Here-blue?link=https%3A%2F%2Fraw.githubusercontent.com%2Fsentrysoftware%2FstudioX-templates%2Frefs%2Fheads%2Fmaster%2Ftemplates%2FairVisual%2FairVisual.cfg)      ![Static Badge](https://img.shields.io/badge/Need%20Support%3F%20-%20Click%20Here-blue?link=https%3A%2F%2Fwww.sentrysoftware.com%2Fbmc%2Fabout%23section-contact)
+
+> No support is provided for this template unless purchased. Refer to [Support for Monitoring Studio X Templates](../../support.md) for more details.
```

Comment:
For notes, you can you the [below special GitHub syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):

```suggestion
> [!CAUTION]
> No support is provided for this template unless purchased. Refer to [Support for Monitoring Studio X Templates](../../support.md) for more details.
```

This will be rendered as:

> [!CAUTION]
> No support is provided for this template unless purchased. Refer to [Support for Monitoring Studio X Templates](../../support.md) for more details.